src/main/java/com/hubspot/jinjava/JinjavaConfig.java

```diff
@@ -307,7 +309,10 @@ public Builder withMaxMacroRecursionDepth(int maxMacroRecursionDepth) {
     }
 
     public Builder withReadOnlyResolver(boolean readOnlyResolver) {
```

Comment:
I wanted to keep the existing behavior consistent so kept this method in for the builder.